hack/swift/Makefile

```diff
@@ -0,0 +1,77 @@
+.DEFAULT_GOAL: help
+
+# construct containerized azcli command
+KUBECFG = $(HOME)/.kube
+SSH     = $(HOME)/.ssh
+AZCFG   = $(HOME)/.azure
+AZIMG   = mcr.microsoft.com/azure-cli
+AZCLI   = docker run -v $(AZCFG):/root/.azure -v $(KUBECFG):/root/.kube -v $(SSH):/root/.ssh $(AZIMG) az
+
+# overrideable variables
+USER       ?= $(whoami)
+CLUSTER    ?= $(USER)-swift-$(REGION)
+GROUP      ?= $(CLUSTER)
+REGION     ?= centraluseuap
+SUB        ?= $(AZURE_SUBSCRIPTION)
+VNET       ?= $(CLUSTER)
+
+
+##@ Help
+
+help:  ## Display this help
+	@awk 'BEGIN {FS = ":.*##"; printf "\nUsage:\n  make \033[36m<target>\033[0m\n"} /^[a-zA-Z_-]+:.*?##/ { printf "  \033[36m%-15s\033[0m %s\n", $$1, $$2 } /^##@/ { printf "\n\033[1m%s\033[0m\n", substr($$0, 5) } ' $(MAKEFILE_LIST)
+
+
+##@ Utilities
+
+azcfg:
+	@$(AZCLI) account set -s $(SUB)
+	@$(AZCLI) group create --location $(REGION) --name $(GROUP)
+	@$(AZCLI) configure --defaults group=$(GROUP)
+	@$(AZCLI) extension add --name aks-preview --yes
+
+set-kubeconf: ## Adds the kubeconf for $cluster
+	$(AZCLI) aks get-credentials -n $(CLUSTER) -g $(GROUP)
+
+unset-kubeconf: ## Deletes the kubeconf for $cluster
+	@kubectl config unset current-context
+	@kubectl config delete-cluster $(CLUSTER)
+	@kubectl config delete-context $(CLUSTER)
+	@kubectl config delete-user clusterUser_$(CLUSTER)_$(CLUSTER)
+
+shell: ## print $AZCLI so it can be used outside of make
+	@echo $(AZCLI)
+
+
+##@ SWIFT
+
+swift-vars: ## Show the env vars configured for the swift command
+	@echo SUB=$(SUB)
+	@echo GROUP=$(GROUP)
+	@echo REGION=$(REGION)
+	@echo VNET=$(VNET)
+	@echo CLUSTER=$(CLUSTER)
+
+swift-net-up: azcfg ## Create required swift vnet/subnets
+	$(AZCLI) network vnet create -g $(GROUP) --name $(VNET) --address-prefixes 10.0.0.0/8 -o none
```

Comment:
Can we poelase add -l flag to all create commands, sometimes the create commands will use defaults added in azconfig and that might result in errors.